example/instruction_tuning_pipeline/finetune_clm.py

```diff
@@ -757,6 +756,32 @@ def concatenate_data(dataset, max_seq_length):
                     training_args.output_dir, state_dict=unwrapped_model.state_dict()
                 )
 
+    if finetune_args.save_merged_model:
+        if isinstance(model, PeftModel):
+            model = model.merge_and_unload()
+            saved_dir = os.path.join(training_args.output_dir, "merged_model")
+            os.makedirs(saved_dir, exist_ok=True)
+            print(f"copy base model config to {saved_dir}")
+            os.system(f"cp {model_args.model_name_or_path}/* {saved_dir}")
+            print(f"copy merged model code to {saved_dir}")
+            os.system(f"cp {finetune_args.merge_model_code_dir}/* {saved_dir}")
```

Comment:
The merged code only apply to SSF, I think we can move it to below ssf branch. Also please add a check whether the merge code path exists

Reply:
Moved and checked.